examples/translation-jobs.json

```diff
@@ -0,0 +1,89 @@
+{
+    "total_items": 6,
+    "translations": [
+        {
+            "id": "a8fd23a9-55cd-4c88-97db-0174e3d7164d",
+            "item": {
+                "type": "article",
+                "id": "054824",
+                "field": "name-fr",
+                "name": "Feldschlösschen Premium Bier"
+            },
+            "created_on": "2016-10-11T12:11:32+02:00",
+            "created_by": "Ivan Wolf",
+            "source_language": "de",
+            "source_text": "Feldschlösschen Premium Bier",
+            "translated_language": "fr"
```

Comment:
I recommend including all fields in the listing (it's not like there are so many).